Sources/User.php

```diff
@@ -1104,7 +1104,7 @@ public function format(bool $display_custom_fields = false): array
 			$this->formatted['custom_fields'] = [];
 
 			if (!isset(Utils::$context['display_fields'])) {
-				Utils::$context['display_fields'] = Utils::jsonDecode(Config::$modSettings['displayFields'], true);
+				Utils::$context['display_fields'] = Utils::jsonDecode(Config::$modSettings['displayFields'], true) ?? [];
```

Comment:
```suggestion
				Utils::$context['display_fields'] = Utils::jsonDecode(Config::$modSettings['displayFields'] ?? '[]', true);
```

Reply:
I will double-check this one, but it's fine.  I encountered a case where the mod setting was not provided, leading to an error during JSON decoding.  This issue stems from custom profile fields, which I suspect are corrupted on my installation.